src/root/App.js

```diff
@@ -1,8 +1,93 @@
 import React from "react";
-
+import MockData from "../data/mockData.json";
+import LoveIcon from "../images/Love.png";
+import FavoriteIcon from "../images/Favorite.png";
 
 function App() {
-  return <div></div>;
+  return (
+    <div className="col" style={{ display: "flex" }}>
+      <div className="movies-section">
+        <h3 style={{ marginLeft: 68 }}>Movies</h3>
+        {MockData.map((item) => (
+          <div
+            key={item.id}
+            style={{
+              flexDirection: "row",
+              flexWrap: "nowrap",
+              display: "flex",
+              margin: 20,
+              position: "relative",
+            }}
+          >
+            <img
+              src={item.image}
+              alt={item.title}
+              style={{ height: 150, width: 150, marginRight: 10 }}
+            />
+            <div style={{ flex: 1, marginTop: -10 }}>
+              <h5>Name: {item.title}</h5>
+              <h5>Category: {item.categories}</h5>
+              <h5>Year: {item.year}</h5>
+              <h5>Director: {item.director}</h5>
+              <img
+                onClick={() => window.open(item.url, "_blank")}
+                src={FavoriteIcon}
+                alt="Favorite icon"
+                style={{
```

Comment:
Bunu sag yukariya sabitlemek icin yaptiysak gereksiz olmus. div in disina aldigimizda kendisini en saga almasi gerekiyor, eger almazsa 
```javascript
            style={{
              flexDirection: "row",
              flexWrap: "nowrap",
              display: "flex",
              margin: 20,
              position: "relative",
justifyContent: 'space-evenly'
            }}`

justifyContent: 'space-evenly' gibi bir deger isimize yarayabilir kontrol edelim